### ISSUE TITLE
The prediction was completely black

### ISSUE THREAD

**Issue by PinkGhost0812** [milesial/Pytorch-UNet#370]
I used my own dataset to make predictions, the picture size is 200*200 Three-channel and th mask for single channel Only black and white. Trying to adjust the learning rate and epoch did not work. How could I solve the prediction that the result is all black?

**Comment by PinkGhost0812** [comment]
![image](https://user-images.githubusercontent.com/33644927/172015422-b120dd4c-4d45-4075-b28b-f318d33d5b77.png)
But he can run normally and train, that is, the predicted result will be all black

**Comment by cs19469** [comment]
Have you solved it?

**Comment by HaithemH** [comment]
Have the same issue

**Comment by leo1328** [comment]
我也是，你们解决了吗？mask的最大值都是0。。

**Comment by Joycoooi** [comment]
> I used my own dataset to make predictions, the picture size is 200*200 Three-channel and th mask for single channel Only black and white. Trying to adjust the learning rate and epoch did not work. How could I solve the prediction that the result is all black?
Please have you solved this problem, I also encountered this problem,
